webapp-django/crashstats/base/jinja2/crashstats_base.html

```diff
@@ -176,8 +176,7 @@ <h1>Product Navigation</h1>
                     {% csrf_token %}
                     <div
                         id="signin2"
-                        class="google-signin"
-                        style="float:right"
+                        class="google-signin float_right"
```

Comment:
Could we maybe just apply the float to the `google-signin` class, or add a different class? `float_right` isn't a terribly semantic classname.